ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -1120,4 +1120,38 @@ extension RCTConvert {
 #endif
         return pictureInPictureConfig
     }
+
+    /**
+     Utility method to instantiate a `UiConfig` from a JS object.
+     - Parameter json: JS object
+     - Returns: The produced `UiConfig` object
+     */
+    static func playerViewConfig(_ json: Any?) -> RNPlayerViewConfig? {
+        guard let json = json as? [String: Any?],
+              let uiConfigJson = json["uiConfig"] as? [String: Any?] else {
+            return nil
+        }
+
+        return RNPlayerViewConfig(
+            uiConfig: UiConfig(
+            playbackSpeedSelectionEnabled: uiConfigJson["playbackSpeedSelectionEnabled"] as? Bool ?? true
+            )
+        )
```

Comment:
Minor indentation problem here:

```suggestion
        return RNPlayerViewConfig(
            uiConfig: UiConfig(
                playbackSpeedSelectionEnabled: uiConfigJson["playbackSpeedSelectionEnabled"] as? Bool ?? true
            )
        )
```

Reply:
Interesting, I would expect the linter takes care of that . 🤔

Reply:
I think we don't have the indentation rule disabled because there is a bug in the linter for that.